lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -66,9 +66,12 @@ def initialize_passive_dispatcher
     self.waiters    = []
     self.alive      = true
 
+    # Ensure that there is only one leading and trailing slash on the URI
+    resource_uri = "/" + self.conn_id.to_s.gsub(/(^\/|\/$)/, '') + "/"
```

Comment:
Given that it's possible for Base64 encoded stuff to include slashes, and they _could_ appear at the end, should we do something to protect against this? eg.:
``` ruby
2.1.5 :002 > puts Base64.encode64("subjects?")
c3ViamVjdHM/
```
I realise it's a pretty slim chance, but should we care?